### PR TITLE
ibrahimayodotun

### PR DESCRIPTION
**Pull request by 01dotun**

### CODE REVIEW

File: myscript.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = "Ibrahim Ayodotun"
+hng_id= "HNG-01955"
+email="ibrahimayodotun@gmail.com"
+print(f"Hello World " + "this is " + name  + " with " + "HNGi7 ID "+hng_id + " and email "+ email+ " using "+ "python language" + " for stage 2 task_.")
```
Comment (elizabethadegbaju) [inline myscript.py:4]: Please delete this file.

File: scripts/myscript.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name = "Ibrahim Ayodotun"
+hng_id= "HNG-01955"
+email="ibrahimayodotun@gmail.com"
+print(f"Hello World " + "this is " + name  + " with " + "HNGi7 ID "+hng_id + " and email "+ email+ " using "+ "python language" + " for stage 2 task_.")
```
Comment (elizabethadegbaju) [inline scripts/myscript.py:4]: Please delete this file also. You need just one file

**Comment by 01dotun** [comment]
Good day,
I have deleted the file.
Thank you.

File: myscript.py (hunk 1 above)
Comment (elizabethadegbaju) [reply myscript.py]: > Please delete this file.

remains this one. you need only one file which is located in scripts folder and you added three so it is two you need to delete.

**Pull request merged**